src/main/java/persistence/association/EntityOneToManyPath.java

```diff
@@ -0,0 +1,42 @@
+package persistence.association;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import persistence.meta.EntityMeta;
+
+public class EntityOneToManyPath {
+    private static final int START_PATH_LEVEL = 0;
```

Comment:
```suggestion
    private static final int START_PATH_LEVEL = 0;
    
```

상수와 상태값을 구분지어 주는 것도 좋습니다!

Reply:
각 컨텍스트를 분리하기 위한 띄어쓰기! 주의하겠습니다!